app/views/root/_gem_base.html.erb

```diff
@@ -16,15 +20,17 @@
       short_description: @emergency_banner.short_description,
     }
   end
-
-  user_satisfaction_survey = '<div id="user-satisfaction-survey-container"></div>'
-  global_bar = render "components/global_bar"
+  global_bar = ''
+  user_satisfaction_survey = '<div id="user-satisfaction-survey-container"></div>' unless omit_user_satisfaction_survey
+  global_banner = render "components/global_bar" unless omit_global_banner
+  global_bar << user_satisfaction_survey if user_satisfaction_survey
```

Comment:
Could line 24 be combined with this line for simplicity? e.g.

```
global_bar << '<div id="user-satisfaction-survey-container"></div>' unless omit_user_satisfaction_survey
```

Reply:
Have done, thank you